config/locales/en.yml

```diff
@@ -447,7 +447,10 @@ en:
     join: Join
     join_course: Join course
     join_prompt: Join '%{title}'?
-    join_successful: You’ve successfully joined %{title}.
+    join_successful: >
+     You’ve successfully joined %{title}.
+     In case of an Editathon, the dashboard updates itself about every 5 minutes or so.
+     For other course levels, you could always refresh this page to get the latest data.
```

Comment:
You’ve successfully joined %{title}.

The dashboard will automatically keep track of your contributions with periodic updates. Check back later to see what impact you've made!